base/range.jl

```diff
@@ -367,6 +370,26 @@ julia> LinRange(1.5, 5.5, 9)
 9-element LinRange{Float64}:
  1.5,2.0,2.5,3.0,3.5,4.0,4.5,5.0,5.5
 ```
+
+Compared to [`range`](@ref), `LinRange` should have less overhead but won't try to correct
+for floating point errors:
```

Comment:
```suggestion
Compared to [`range`](@ref), direct `LinRange` construction should have less overhead
but won't try to correct for floating point errors:
```
(The previous sentence implied `LinRange` is not one of the products of `range` calls.)

Reply:
I made a change with slightly different wording than you suggest. Can you let me know what you think?

Reply:
Great, thx for checking out my thoughts

Reply:
😄